src/internal/tester/integration_runners.go

```diff
@@ -11,7 +11,10 @@ import (
 )
 
 const (
-	CorsoLoadTests                                = "CORSO_LOAD_TESTS"
+	CorsoLoadTests = "CORSO_LOAD_TESTS"
+	// Run only integration tests.
+	CorsoIntegrationTests = "CORSO_INTEGRATION_TESTS"
+	// Run all tests, unit and integration.
 	CorsoCITests                                  = "CORSO_CI_TESTS"
```

Comment:
IMO: `Integration` should be 1:1 with `CI`.  We should have one flag or the other, but not both.